pkg/condition/conditions.go

```diff
@@ -14,17 +14,25 @@
 package condition
 
 import (
-	"github.com/go-logr/logr"
+	"strconv"
 	"time"
+
+	"github.com/aws/amazon-vpc-resource-controller-k8s/pkg/k8s"
+	"github.com/go-logr/logr"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
 type condition struct {
 	hasDataStoreSynced *bool
 	log                logr.Logger
+	K8sAPI             k8s.K8sWrapper
 }
 
 const CheckDataStoreSyncedInterval = time.Second * 10
 
+var enableWindowsIPAMKey = "enable-windows-ipam"
```

Comment:
I think it's being reused at other places too, can we declare this as constant in config.go